resources/dicts/thoughts/dead/darkforest/exiled.json

```
@@ -1,3 +1,63 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "id": "exiled_df_dead_thoughts",
+    "thoughts": [
+      "Is thinking of the dissaproving glares of {PRONOUN/m_c/poss} old Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} train",
+        "Is watching {PRONOUN/m_c/poss} old Clan with scorn hoping for an opportunity to get back at them",
+        "Wants to make {PRONOUN/m_c/poss} old leader pay for exiling {PRONOUN/m_c/object}",
+        "Is planning on making c_n pay for exiling {PRONOUN/m_c/object}"
+        ]
+    },
+    {
+        "id": "exiled_df_evil",
+        "thoughts": [
+            "Is imagining the tree {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} clawing at is an old Clanmate",
+            "Trains extra hard waiting for an opportunity to get {PRONOUN/m_c/poss} revenge",
```

```suggestion
            "Trains extra hard, waiting for an opportunity to get {PRONOUN/m_c/poss} revenge",
```
this one is a tentative suggestion, either way works okay